src/main/java/com/iexec/core/task/TaskService.java

```diff
@@ -432,6 +434,63 @@ private void initializedOrRunning2ContributionTimeout(Task task) {
         }
     }
 
+    /**
+     * When all workers have failed to run the task, we send the new status as soon as possible.
+     * It avoids a state within which we know the task won't be updated anymore, but we wait for a timeout.
+     * <br>
+     * We consider that all workers are in a `RUNNING_FAILED` status when:
+     * <ol>
+     *     <li>All alive workers have tried to run the task;</li>
+     *     <li>All workers that have tried to run the task have failed, whether they are alive or lost;</li>
+     *     <li>No replicate has reached the `COMPUTED` status.</li>
+     * </ol>
+     *
+     * @param task Task to check and to make become {@link TaskStatus#RUNNING_FAILED}.
+     */
+    private void running2RunningFailed(Task task) {
+        boolean isRunningTask = task.getCurrentStatus().equals(RUNNING);
+        if (!isRunningTask) {
+            return;
+        }
+
+        final List<Replicate> replicates = replicatesService.getReplicates(task.getChainTaskId());
+
+        final List<Worker> aliveWorkers = workerService.getAliveWorkers();
+
+        // If an alive worker has not run the task, it is not a `RUNNING_FAILURE`.
+        final Predicate<Worker> hasWorkerAlreadyTried = worker -> replicates
```

Comment:
As we are dealing with failures, I find `try` to be more appropriate than `participate`. The second one tends to imply the workers have solved successfully the task while the first one tends to imply the workers may have failed after some tries.
If consistency is crucial there, we could rename the variable. Otherwise, I'd prefer to keep the current one.